wee_slack.py

```diff
@@ -2199,6 +2203,8 @@ def remove_reaction(self, reaction, user):
         else:
             pass
 
+    def has_mention(self):
+        return any(word in self.message_json.get('text') for word in self.channel.mentions())
```

Comment:
You can use `w.string_has_highlight` to check if the string matches.

https://weechat.org/files/doc/stable/weechat_plugin_api.en.html#_string_has_highlight

Reply:
✓